client/src/java/com/zimbra/client/ZContact.java

```diff
@@ -96,10 +97,12 @@ public long getLength() {
 
     /* Note for contacts, use a more limited set of flags than in ZItem */
     public enum Flag {
-        flagged('f'),
-        attachment('a');
+        flagged(-6, 'f'),
+        attachment(-2, 'a');
 
         private char mFlagChar;
+        final int id;
+        final int bitmask;
```

Comment:
I removed the ZContact.Flag commit and changed ZContactHit::getFlagBitmask to use ZItem::Flag::toBitmask